### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by edvinalestig**
Hello, I found some bugs so I decided to fix them. The check you added for the button in the setting menu isn't necessary because IPT2 is not required for all the functionality, only the people who got on/off at the last stop. Without IPT2 it still shows the passengers onboard which the setting disables.

A bug I found is that the last stop stats did not show up when the vehicle was reversed, so I fixed it. I believe it should all be ready for release.

### CODE REVIEW

File: FPSCamera/InstanceCameras/VehicleCamera.cs [hunk 1]
```diff
@@ -173,8 +173,9 @@ public void GetInstanceSpeed(Vector3 position)
 
         public void GetLastStopExchange()
         {
-            int lastGone = ImprovedPublicTransport2.Detour.VehicleManagerMod.m_cachedVehicleData[(int)followInstance].LastStopGonePassengers;
-            int lastNew = ImprovedPublicTransport2.Detour.VehicleManagerMod.m_cachedVehicleData[(int)followInstance].LastStopNewPassengers;
+            ushort firstVehicle = vManager.m_vehicles.m_buffer[(int)followInstance].GetFirstVehicle(followInstance);
```
Comment (tony56a) [inline FPSCamera/InstanceCameras/VehicleCamera.cs:176]: Does IPT2 only cache the first vehicle for trains? I noticed that the vanilla game will [leave a train reversed when going the other way](https://github.com/tony56a/Skylines-FPSCamera/blob/5399081a229ff7f5dfed933be3ba15e98962cf3a/FPSCamera/InstanceCameras/VehicleCamera.cs#L58)

**Review by tony56a — COMMENTED** [review]
Thanks for correcting the UI issue! Just 1 quick question about the other bug.

File: FPSCamera/InstanceCameras/VehicleCamera.cs (hunk 1 above)
Comment (edvinalestig) [reply FPSCamera/InstanceCameras/VehicleCamera.cs:176]: Apparently. Before I added the ```firstVehicle``` it always showed ```-0 | +0``` when reversed. But now it works.

Comment (tony56a) [reply FPSCamera/InstanceCameras/VehicleCamera.cs:176]: Sounds good if this was verified for trains. I'll apply the changes and publish the mod in a few hours then.

**Pull request merged**